cirq-web/cirq_web/widget_test.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cirq_web
+
```

Comment:
we need a bit more tests as we move things to Widget: create a subclass test widget and verify the behaviors - you can probably move a bunch of the test logic you're currently using for BlochSphere.